### ISSUE TITLE
Discontinued? 

### ISSUE THREAD

**Issue by Megatronicus** [ahsanalidev/flutter_nsfw#10]
I see on pubdev that this code is discontinued? 

https://pub.dev/packages/flutter_nsfw

Is there another option or a way to filter nsfw images like this. This seems like a great package, I did some examples two years ago and it was really simple to use, now I see they've changed the library, still don't support web and the whole code is changed so all working flutter examples I made with tflite are now obsolete. Have you maybe made a replacement library, or do you know an example of one like this to filter nsfw images?

Thank you in advance.

**Comment by Megatronicus** [comment]
Can you please answer?

**Comment by ahsanalidev** [comment]
Yeah, it was working fine on images, but for video NSFW detection, one of the dependencies that I was using on Android was deprecated, and I didn't have time to fix that, so I discontinued this package.

**Comment by ahsanalidev** [comment]
if you just want to detect NSFW you can use google cloud apis they also provide nudity detection functionality
